pkg/controller/upload-controller.go

```diff
@@ -476,7 +476,12 @@ func (c *UploadController) getOrCreateUploadPod(pvc *v1.PersistentVolumeClaim, p
 			ClientName:     clientName,
 		}
 
-		pod, err = CreateUploadPod(args)
+		podResourceRequirements, err := GetDefaultPodResourceRequirements(c.cdiClient)
```

Comment:
Same as clone controller, I think it makes sense to look up the pod requirements in the function that builds the spec.

Reply:
DONE